kolibri/plugins/coach/assets/src/vue/groups-page/group-section.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+
+  <div class="group-section">
+    <h2>{{ group.name }}</h2>
+    <span>{{ $tr('numLearners', {count: group.users.length }) }}</span>
+    <span v-if="group.users.length">{{ `${selectedUsers.length} ${$tr('selected')}` }}</span>
+    <icon-button :text="$tr('moveLearners')"
+      :primary="true"
+      size="small"
+      @click="emitMove"
+      :disabled="selectedUsers.length === 0" />
+    <ui-button v-if="!isUngrouped"
+      color="primary"
+      :has-dropdown="true"
+      ref="dropdownButton"
+      size="small">
+      <ui-menu slot="dropdown"
```

Comment:
The new Keen-ui component. Pulling this PR in a bit to see how it reacts in mobile contexts